src/index.tsx

```diff
@@ -32,7 +34,7 @@ i18next
     interpolation: {
       escapeValue: false
     },
-    lng: 'en',
+    lng: storage.getItem('language') || 'en',
```

Comment:
when selecting a language other than `en`, the app was loaded in english for a moment and then changed to the user's language, which looked weird and triggered an extra re-render